plugins/design-system/components/component-example/CodeExampleFooter.js

```diff
@@ -0,0 +1,73 @@
+import classNames from "classnames/dedupe";
+import React, { Component } from "react";
+
+const defaultClasses = {
```

Comment:
So `CodeExampleFooter` and `CodeExampleHeader` are essentially the same only `defaultClasses` are different. Thinking if it is a ok duplication or not.